### PR TITLE
option to disable success notification

### LINKED ISSUE #9344: create easy API to disable toast from transaction
for example
with the new create collection modal #9333
the toast becomes useless and not slick ui

image

we should disable it

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Context

- [x] Closes #9344

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

### CODE REVIEW

File: composables/useTransaction.ts [hunk 1]
```diff
@@ -40,6 +40,10 @@ import {
 import { ApiPromise } from '@polkadot/api'
 import { isActionValid } from './transaction/utils'
 
+export type TransactionOptions = {
+  disableNSuccessNotification?: boolean
```
Comment (hassnian) [inline composables/useTransaction.ts:44]: typo ?
CleanShot 2024-02-10 at 11 26 33@2x

Comment (daiagi) [reply composables/useTransaction.ts]: oh yeah :+1:

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 7.66 USD/DOT ~ 3.916 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0xd1af2ea7a6f48f40532a4d1e9f3b61ac39c5399417846bcac8057a0a2b3a0afd](https://polkadot.subscan.io/extrinsic/0xd1af2ea7a6f48f40532a4d1e9f3b61ac39c5399417846bcac8057a0a2b3a0afd)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**